plugins/blip/basics/components/chart/WrapCount.js

```diff
@@ -39,7 +38,11 @@ var WrapCount = React.createClass({
     );
   },
   generateDots: function(start, end, dotSize, pad) {
-    pad = pad || 1.5;
+    pad = pad || 0;
```

Comment:
These edits seem like they may not be entirely settled, between the commented code and the immediate reassignment of `pad`.

Reply:
Yeah, thanks.  I'll clean this up.